src/routes/Dataset/components/DatasetView/DatasetView.js

```diff
@@ -72,12 +72,25 @@ class DatasetView extends React.PureComponent {
   render() {
     const { dataset, publication, dataGouvDataset, fetchGeoJson, t, i18n } = this.props
     const status = dataset.metadata.status
+    let thumbnailUrl
+
+    if (dataset.metadata.thumbnails && dataset.metadata.thumbnails.length) {
+      thumbnailUrl = `https://inspire.data.gouv.fr/api/geogw/records/${dataset.recordId}/thumbnails/${dataset.metadata.thumbnails[0].originalUrlHash}`
```

Comment:
it’d make more sense so have something like 

```jsx
const hasThumbnails = dataset.metadata.thumbnails && dataset.metadata.thumbnails.length > 0
```

The thumbnailUrl check for the `<DatasetBlock />` doesn’t make a lot of sense